core/scripts/cli/voting/commitVotes.js

```diff
@@ -0,0 +1,108 @@
+const inquirer = require("inquirer");
+const style = require("../textStyle");
+const getDefaultAccount = require("../wallet/getDefaultAccount");
+const filterRequests = require("./filterRequestsByRound");
+const { VotePhasesEnum } = require("../../../../common/Enums");
+const { constructCommitment } = require("../../../../common/VotingUtils");
+const batchCommitVotes = require("./batchCommitVotes");
+
+module.exports = async (web3, voting) => {
+  style.spinnerReadingContracts.start();
+  const pendingRequests = await voting.getPendingRequests();
+  const roundId = await voting.getCurrentRoundId();
+  const roundPhase = await voting.getVotePhase();
+  const account = await getDefaultAccount(web3);
+  const filteredRequests = await filterRequests(pendingRequests, account, roundId, roundPhase, voting);
+  style.spinnerReadingContracts.stop();
+
+  if (roundPhase.toString() === VotePhasesEnum.REVEAL) {
+    console.log(
+      `The current vote phase is the "reveal" phase; in the reveal phase, you can only reveal already committed votes. You cannot vote on price requests in this phase.`
+    );
+  } else if (filteredRequests.length === 0) {
+    console.log(`No pending price requests to commit votes for!`);
+  } else {
+    console.group(`${style.instruction(`\nPlease select which price requests you would like to commit votes for`)}`);
+
+    // To display properly, give each request a 'value' parameter
+    for (let i = 0; i < filteredRequests.length; i++) {
+      let request = filteredRequests[i];
+      request.value = `${web3.utils.hexToUtf8(request.identifier)} @ ${style.formatSecondsToUtc(
+        parseInt(request.time)
+      )}`;
+    }
+
+    const checkbox = await inquirer.prompt({
+      type: "checkbox",
+      name: "requestsCheckbox",
+      message: `After selecting which requests you want to vote on, you will be prompted to manually enter in a price for each request. You can change these votes later.`,
+      choices: filteredRequests
+    });
+    if (checkbox["requestsCheckbox"]) {
+      console.log(
+        style.instruction(
+          `Next, enter prices for the selected requests. Prices must be positive numbers, invalid input will default to 0.`
+        )
+      );
+
+      const newCommitments = [];
+      const failures = [];
+
+      // Prompt user to enter a price per vote construct commitments for the votes
+      const selections = checkbox["requestsCheckbox"];
+      for (let i = 0; i < selections.length; i++) {
+        // Prompt user to enter a price per vote and commit the votes
+        const priceInput = await inquirer.prompt({
+          type: "number",
+          name: "price",
+          default: 0,
```

Comment:
yeah without the default, `inquirer` doesn't play nicely. It will display an error message and then put a `NaN` in the user's type box which for some reason I cannot backspace/delete